webpack/assets/javascripts/react_app/components/Pagination/index.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useHistory } from 'react-router-dom';
+import URI from 'urijs';
+import classNames from 'classnames';
+import {
+  Pagination as PF4Pagination,
+  PaginationVariant,
+} from '@patternfly/react-core';
+import { translate as __ } from '../../common/I18n';
+import {
+  getURIpage,
+  getURIperPage,
+  changeQuery,
+} from '../../common/urlHelpers';
+import { useForemanSettings } from '../../Root/Context/ForemanContext';
+import './index.scss';
+
+const Pagination = ({
+  itemCount,
+  onSetPage,
+  onPerPageSelect,
+  className,
+  page: propsPage,
+  perPage: propsPerPage,
+  noSidePadding,
+  ...props
+}) => {
+  const { perPage: settingsPerPage } = useForemanSettings();
+  const history = useHistory();
+  const page = getURIpage() || propsPage;
+  const urlPerPage = getURIperPage();
+  const perPage = urlPerPage || propsPerPage || settingsPerPage;
+
+  const paginationTitles = {
+    items: __('items'),
+    page: __('page'),
```

Comment:
pf4 adds `s` to the word `page` which doesn't work well with other languages, can we leave `page` as an empty string? in the previous pagination there was no word used for page 
![Screenshot from 2021-12-17 11-52-51](https://user-images.githubusercontent.com/30431079/146533965-9a4539fa-259e-49f5-9419-926c2161bdb6.png)

Reply:
another option is to pass `page: __('pages'),` until PF will add a plural option in titles

Reply:
But then it will be `pages`s with 2 `s`

Reply:
right, looks like @boaz0 is fixing it for us on PF: https://github.com/patternfly/patternfly-react/pull/6712
Thank you @boaz0 🥳

Reply:
removing `page` from the list for now

Reply:
@MariaAga the workaround @Ron-Lavi suggested sounds good to me. Just check that the `itemCount > perPage` to determine whether to use plural or singular form and pass It to `title.pages`.